app/DAO/BakeryDB.scala

```diff
@@ -0,0 +1,55 @@
+package DAO
+
+import doobie._
+import doobie.implicits._
+import doobie.implicits.javasql._
+import cats.effect.IO
+import doobie.util.ExecutionContexts
+import cats.implicits
+import doobie.postgres.implicits._
+import doobie.postgres.pgisimplicits._
+import cats.effect.unsafe.implicits.global
+import java.time.{LocalDateTime, OffsetDateTime}
+
+class BakeryDB {
+
+  val xa = Transactor.fromDriverManager[IO](
+    "org.postgresql.Driver",
+    "jdbc:postgresql://localhost:5500/bakerydb",
+    "postgres",
+    "pass"
+  )
+
+  case class Product(
+      id: String,
+      name: String,
+      quantity: Int,
+      price: Double,
+      createdAt: OffsetDateTime,
+      updatedAt: OffsetDateTime
+  )
```

Comment:
extract the case class out of this class or into a file that is called something like `DBModels`